data/ocr/store_regex.txt

```diff
@@ -10,6 +10,15 @@ Aldi||gold[aä]hren
 Aldi||gutbio
 Aldi||milsani
 Aldi||sprengel
+Aldi||liveGfree
+Aldi||Simply Nature
+Aldi||Specially Selected
+Aldi||Little Journey
```

Comment:
Same thing for little journey or friendly farms

Reply:
those combinations of words are unlikely to happen ? I'm removing simply nature

Reply:
Well technically they can. The thing is, it is autoapplied here. We should either check that it doesn't occur in the dataset, or have a way to have validation from a human.